src/alignment.jl

```diff
@@ -100,41 +104,74 @@ function Base.show(io::IO, aln::Alignment)
       print(io, "  CIGAR string: ", cigar(aln))
 end
 
-"""
-    seq2ref(aln::Alignment, i::Integer)::Tuple{Int,Operation}
-
-Map a position `i` from sequence to reference.
-"""
-function seq2ref(aln::Alignment, i::Integer)::Tuple{Int,Operation}
-    idx = findanchor(aln, i, Val{true})
```

Comment:
I was never sure whether `Val(...)` generates a code to allocate that object or not, so to avoid allocation I was using type approach, but I'll change it to `Val()`.

Reply:
There is no allocation - the two things are completely equivalent. It's just the norm to use instances instead of types whenever you have a zero-field struct. (e.g. Julia's Base and the rest of BioJulia does it).

Reply:
Actually, that was not my code. %) In the refactored functions `Val` is replaced by passing function references.